Update Continuous Testing global configuration docs (#15992)
DataDog/documentation: content/en/continuous_testing/troubleshooting.md

## Terminology

CI batch 
: The group of Continuous Testing tests triggered through a continuous integration or continuous delivery (CI/CD) pipeline or the [Datadog Synthetic Monitoring API][2].

Test run

> : A single run of a Continuous Testing test. This can be [API][7] or [browser tests][8]. If you have configured retries, they count as individual test runs. For example, a test with two retries can have up to three associated test runs.

Reviewer: ```suggestion
: A single run of a Continuous Testing test, which can be an [API][7] or [browser test][8]. If you have configured retries, they count as individual test runs. For example, a test with two retries can have up to three associated test runs.
```

Parallel test

> : A test that is run at the same time as another test. Parallel tests are part of the Continuous Testing product. In order to control how many tests you would like to run in parallel, configure parallelization in the [settings page][9].

Reviewer: Trying to rephrase "Parallel tests are part of the Continuous Testing product" in the first sentence.

```suggestion
: A Continuous Testing test that is run at the same time as another Continuous Testing test. To set how many tests you would like to run in parallel, configure parallelization on the [Continuous Testing Settings page][9].
```

Check whether you are using API endpoints to trigger your CI/CD test runs. In order to populate the CI Results Explorer with CI metadata, you must use one of Datadog's [native integrations][5], or the [NPM package][6].

## Within your CI/CD pipeline

> ### My tests are timing out in my CI. What do I do?

Reviewer: ```suggestion
### My tests are timing out in my CI pipeline
```

> The first thing to check is which failure mode flags you are passing in your [global configuration file][10]. For CI runs that contain multiple tests, some tests will be queued based on the parallelization setting defined in your Continuous Testing [settings page][9]. You may need to adapt both your configuration and your parallelization based on your organisational needs.

Reviewer: ```suggestion
The first thing to check is which failure mode flags you are passing in your [global configuration file][10]. For CI runs that contain multiple tests, some tests are queued based on the parallelization setting defined on the [Continuous Testing Settings page][9]. You may need to adapt both your configuration and parallelization based on your organizational needs.
```
